cli/dcoscli/node/main.py

```diff
@@ -4,18 +4,29 @@
     dcos node --info
     dcos node [--json]
     dcos node log [--follow --lines=N --master --slave=<slave-id>]
+    dcos node ssh [--option SSHOPT=VAL ...]
```

Comment:
mkay. why do we have --option and SSHOPT? I'm thinking it should be more analogous to how we specify multiple options in marathon `dcos marathon app update [--force] <app-id> [<properties>...]`